packages/config/config/devices/templates/master_template.json

```diff
@@ -78,6 +100,26 @@
 		"writeOnly": false,
 		"allowManualEntry": true
 	},
+	"base_0-180_nounit": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 180,
+		"defaultValue": 30,
```

Comment:
This default value seems a bit arbitrary for a master template IMO.
I'm sure there are many other devices where a parameter has a range of 0-180 with a different default value.